tables/rx_potency/atmar21-pot.csv

```diff
@@ -0,0 +1,151 @@
+﻿ref_name,rx_name,iso_name,section,assay_name,potency_type,potency,cumulative_count,potency_upper_limit,potency_lower_limit,potency_unit,date_added
+Atmar21,Ad26COV2S_2dose_mRNA1273_1dose_1day,WA1 Spike,Table 2,Pseudovirus (lentivirus),NT50,57,51,45118,10,NULL,2022-06-21

```

Comment:
The EUA standard of Ad26.COV.2 is 1 dose.

Reply:
Ah, I'll change this then

Reply:
Are the changes around this correct?